Documentation/gettingstarted/k8s-install-kubeadm.rst

```diff
@@ -4,8 +4,49 @@
     Please use the official rendered version released here:
     https://docs.cilium.io
 
+**************************
 Installation using kubeadm
-==========================
+**************************
 
-Instructions about installing Cilium on Kubernetes cluster deployed by kubeadm
-are available in the `official Kubernetes documentation <https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#pod-network>`_.
+This guide describes deploying Cilium on the Kubernetes cluster created with
+``kubeadm``.
+
+For installing ``kubeadm`` on your system, please refer to `the official
+kubeadm documentation
+<https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/>`_
+The official documentation also describes additional options of kubeadm which
+are not mentioned here.
+
+If you are interested in using Cilium's kube-proxy replacement, please
+follow the :ref:`kubeproxy-free` guide and skip this one.
+
+Create the cluster
+==================
+
+Initialize the control plane via executing on it:
+
+.. code:: bash
+
+  kubeadm init
```

Comment:
Maybe you could mention in a note that to deploy w/o kube-proxy, you can use `--skip-phases=kube-proxy`, and then add a link to https://docs.cilium.io/en/v1.8/gettingstarted/kubeproxy-free/ for more info?

Reply:
Done